addons/mail/static/src/core/common/message_reactions.xml

```diff
@@ -7,17 +7,17 @@
             'ms-3': !(env.inChatWindow and env.alignedRight) and (props.message.is_discussion),
         }"
         t-attf-class="{{ props.message.is_discussion ? 'mt-n2' : 'mt-1' }}">
-        <button t-foreach="props.message.reactions" t-as="reaction" t-key="reaction.content" class="o-mail-MessageReaction btn d-flex p-0 border rounded-1 mb-1"
+        <button t-foreach="props.message.reactions" t-as="reaction" t-key="reaction.content" class="o-mail-MessageReaction btn d-flex p-0 rounded-3 mb-1 fs-5"
```

Comment:
`rounded-3` look a bit much looking at the screenshots, maybe just use `rounded-2`?

Reply:
Comparison between `rounded-2` and `rounded-3`:
<img width="897" alt="Screenshot 2024-06-18 at 16 29 52" src="https://github.com/odoo/odoo/assets/6569390/bbd65cac-3a55-4a58-9b5e-aa4eabe8d850">

Rounder is usually more pleasing and also the rounded corners of bubbles are also `rounded-3` so it looks consistent. `rounded-2` is decent but I still have a preference for `rounded-3` if I had to choose.

Reply:
Agree that's essentially subjective.
Though in an attempt to give objective arguments: `rounded-3` is more consistent with roundness of bubble